src/lib/constants.ts

```diff
@@ -3,7 +3,10 @@ import { DeepgramClientOptions } from "./types/DeepgramClientOptions";
 import { FetchOptions } from "./types/Fetch";
 import { version } from "./version";
 
-export const NODE_VERSION = process.versions.node;
+export const NODE_VERSION =
```

Comment:
This will potentially affect how we trace SDK version usage on our API. You should look into the vite process plugin to enable process.versions.node instead.

Reply:
@lukeocodes What's the point of tracing a nodejs version from an environment which does not even have nodejs (only during the build step)? It would make a lot more sense to look at the browser version and environment than nodejs.

Reply:
you're assuming that this only runs in the browser